test/TestProjects/MgmtTypeSpec/src/Generated/FooResource.cs

```diff
@@ -264,7 +264,7 @@ public virtual ArmOperation Delete(WaitUntil waitUntil, CancellationToken cancel
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>Foo_CreateOrUpdate</description>
+        /// <description>Foos_createOrUpdate</description>
```

Comment:
original name is `createOrUpdate`: https://github.com/Azure/autorest.csharp/blob/371c2a2029ae8a2baef218b9b55798433f4b7b75/test/TestProjects/MgmtTypeSpec/foo.tsp#L37